LDAR_Sim/src/ldar_sim_run.py

```diff
@@ -302,3 +303,4 @@ def simulate(
             # -- Batch Report --
             print(f"...Cleaning up batch {batch_count} data")
             concat_output_data(out_dir, batch_count != 0)
+    gen_cross_program_summary_plots(out_dir, base_program)
```

Comment:
Could add in another print here to indicate the cross program plots are being generated here.

Reply:
Will do